src/freenode.adb

```diff
@@ -115,13 +107,14 @@ procedure Freenode is
     Not_Enough_Arguments: exception;
 
 begin
-    if Argument_Count < 1 then
+    if Argument_Count not in Positive then
```

Comment:
I should use natural language constructs like this more often.

Reply:
It is **very** useful, especially to force your thought to sets, which helps preclude the off-by-one error.